guide/raids/amirdrassil.md

```diff
@@ -0,0 +1,445 @@
+
+#### Defensives:
+* Use {{ site.data.talent.as }} or healing consumables during {{ site.data.raid.amirdrassil.raging_inferno }} and when targeted by {{ site.data.raid.amirdrassil.ashen_devastation }} (Mythic-only).
+* Use {{ site.data.talent.ag }} during {{ site.data.raid.amirdrassil.raging_inferno }} and when affected by {{ site.data.raid.amirdrassil.flash_fire }}.
+* If talented, use {{ site.data.talent.earth_shield }} and {{ site.data.talent.healing_stream_totem }} as a movement filler throughout the fight.
+* Use {{ site.data.spell.healing_surge }} if necessary.
+
+#### Utilities:
+* Use {{ site.data.talent.wrt }} to help players affected by {{ site.data.raid.amirdrassil.dream_blossom }} dousing {{ site.data.raid.amirdrassil.burning_ground }} as fast as possible.
+* Use {{ site.data.talent.spirit_walk }} to free yourself from {{ site.data.raid.amirdrassil.scorching_bramblethorn }}.
+* Use {{ site.data.spell.wind_shear }} or {{ site.data.spell.capacitor_totem }} to interrupt {{ site.data.raid.amirdrassil.fiery_flourish }} from {{ site.data.raid.amirdrassil.fiery_treant }}.
+
+</div>
+</div>
+</div>
+
+<div class="card">
+<div class="card-header" id="nymue">
+<div data-toggle="collapse" data-target="#nymue-collapse" aria-expanded="true" aria-controls="nymue-collapse" class="raid-header amirdrassil_the_dreams_hope6"><h2>Nymue, Weaver of the Cycle</h2></div>
+</div>
+<div id="nymue-collapse" class="collapse" aria-labelledby="nymue" data-parent="#accordion">
+<div class="card-body" markdown="1">
+
+## Boss Fight Profile: Single Target
+
+### Talents
+
+#### Lightning T30:
+<div class="iframe-holder">
+<iframe src="https://www.raidbots.com/simbot/render/talents/BYQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAoVSSLJUSSLRDRJQiEAAAAAgSAJlkgmASLJpdACIJBBC?width=530&level=70" frameborder="0" width="530px" height="100%">
+</iframe>
+</div>
+
+#### Fire T31:
+<div class="iframe-holder">
+<iframe src="https://www.raidbots.com/simbot/render/talents/BYQAAAAAAAAAAAAAAAAAAAAAAAAAAAAg0SrUSCgDIlk0oElUSCRAAAAAAKBkUSCaCItkkGCIJJEI?width=530&level=70" frameborder="0" width="530px" height="100%">
+</iframe>
+</div>
+
+<hr>
+
+## Fight Breakdown
+
+#### Damage:
+* Focus down {{ site.data.raid.amirdrassil.cycle_warden }} and {{ site.data.raid.amirdrassil.manifested_dream }} (Mythic-only) whenever they are active.
+* Use {{ site.data.talent.swg }} and {{ site.data.talent.gow }} to maintain uptime whilst being targeted by {{ site.data.raid.amirdrassil.threads_of_life }} or dodging {{ site.data.raid.amirdrassil.lucid_exhaust }} (Mythic-only).
+
+#### Defensives:
+* Use {{ site.data.talent.as }} or healing consumables for overlaps of {{ site.data.raid.amirdrassil.continuum }} and {{ site.data.raid.amirdrassil.lucid_miasma }} (Mythic-only) or when accumulating high stacks of {{ site.data.raid.amirdrassil.verdant_matrix }}.
```

Comment:
site.data.raid.amirdrassil.verdant_matrix does not show on my local version, I believe you didn't include it in the yml file